dom0/sd-dom0-qvm-rpc.sls

```diff
@@ -146,10 +70,23 @@ dom0-rpc-qubes.r5-format-ask-allow:
         securedrop.Log          *           sd-log sd-log deny notify=no
         securedrop.Log          *           @tag:sd-workstation sd-log allow
 
+        securedrop.Proxy        *           sd-app sd-proxy allow
+
+        qubes.Gpg               *           @tag:sd-client sd-gpg allow
```

Comment:
(are we missing the allow rule for GpgImportKey?  `GpgImportKey * @tag:sd-client sd-gpg allow`)